config

```diff
@@ -1,4 +1,4 @@
 [Config]
 location=location_here
 
-metasmoke_host=http://127.0.0.1:3000
+metasmoke_host=dvsbsbsfb
```

Comment:
This line change needs to be reverted. This is a bad placeholder (or reply [here](http://chat.stackexchange.com/transcript/message/29807982#29807982) so we understand)